gcr.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"os/exec"
+	"path"
+	"sort"
+	"strings"
+)
+
+type gcrAuthResponse struct {
+	Token     string `json:"token"`
+	ExpiresIn int    `json:"expires_in"`
+}
+
+type gcrTagsResponse struct {
+	Manifest map[string]json.RawMessage `json:"manifest"`
```

Comment:
With nested struct, you can unmarshal at a time.

```suggestion
	Manifest map[string]json.gcrImageDetail `json:"manifest"`
```

Reply:
Thanks! I fixed it in e977236